apps/nextra/pages/en/build/create-aptos-dapp/templates/custom-indexer.mdx

```diff
@@ -0,0 +1,219 @@
+---
+title: "Create Aptos Dapp Custom Indexer Template"
+---
+
+import { Callout, Steps } from 'nextra/components';
+
+# Create Aptos Dapp Custom Indexer Template
+
+The Custom Indexer template provides a starter dapp with all necessary infrastructure to build a full stack app with custom indexer support.
+
+The Custom Indexer template provides:
+
+- Folder structure - A pre-made dapp folder structure with `src` for frontend, `contract` for Move contract and `indexer` for custom indexer.
+- Dapp infrastructure - All required dependencies a dapp needs to start building on the Aptos network.
+- Wallet Info implementation - Pre-made `WalletInfo` components to demonstrate how one can use to read a connected Wallet info.
+- Message board functionality implementation - Pre-made `MessageBoard` component to create, update and read messages from the Move smart contract.
+- Analytics dashboard - Pre-made `Analytics` component to show the number of messages created and updated.
+- Point program - Minimal example to show you how to define a point program based on events (e.g. create message, update message) and show that on the analytics dashboard, with sorting support.
+
+## Generate the Boilerplate template
+
+On your terminal, navigate to the directory you want to work in and run:
+```bash
+npx create-aptos-dapp@latest
+```
+
+Follow the CLI prompts.
+
+## Getting started
+
+<Steps>
+
+### Publish the contract
+
+Run the below command to publish the contract on-chain:
+```bash
+npm run move:publish
+```
+This command will:
+1. Publish the contract to chain.
+2. Setting the `MODULE_ADDRESS` in the `.env` file to set the contract object address.
+
+### Create a postgres database
+
+Sign up for [Neon postgres](https://neon.tech/) which is our cloud Postgres provider and create a new project. Find the connection string and put it in `DATABASE_URL` in the `.env` file of frontend. 
```

Comment:
where can one find the `connection string`?

Reply:
In neon console